Cargo.toml

```diff
@@ -36,6 +36,7 @@ librespot-connect = { version = "0.4" }
 toml = "0.7"
 color-eyre = "0.6"
 directories = "5.0.1"
+once_cell = "1.20.2"
```

Comment:
In theory, since recently, the standard library also has support for those Lazy cells e.g. with `std::cell::LazyCell`. But to make use of that, we would need a MSRV bump, so this would also be fine for now! Whatever suits you best. :)